src/Pos/Ssc/Extra/Logic.hs

```diff
@@ -73,18 +76,14 @@ sscRunLocalQuery action = do
 sscRunLocalSTM
     :: forall ssc m a.
        (MonadSscMem ssc m, MonadIO m, WithLogger m)
-    => (LoggerNameBox (PureLogger (StateT (SscLocalData ssc) STM)) a) -> m a
+    => NamedPureLogger (StateT (SscLocalData ssc) STM) a -> m a
 sscRunLocalSTM action = do
-    loggerName <- getLoggerName
     localVar <- sscLocal <$> askSscMem
```

Comment:
Ok, but add a comment why this `error` can't be touched please. I took some time for me to understand what's going on.